packages/eslint-plugin/docs/rules/no-unnecessary-type-parameters.mdx

```diff
@@ -76,16 +76,22 @@ function getProperty<T, K extends keyof T>(obj: T, key: K) {
 ## Limitations
 
 Note that this rule allows any type parameter that is used multiple times, even if those uses are via a type argument.
-For example, the following `T` is used multiple times by virtue of being in an `Array`, even though its name only appears once after declaration:
+For example, the following `T` is used multiple times by virtue of being in an `Set`, even though its name only appears once after declaration:
```

Comment:
These no longer need to be changed now that this PR doesn't touch _return_ location arrays. I'll touch these docs up..

```suggestion
For example, the following `T` is used multiple times by virtue of being in an `Array`, even though its name only appears once after declaration:
```